ansible/local-extra-vars.yml

```diff
@@ -0,0 +1,20 @@
+---
+clean_deploy_title: devnet13
+
+block_gas_limit: 50_000_000
+block_time: 5
+chain_id: 2001
+
+polycli_tag: 0.1.20
+edge_tag: v0.9.0
+geth_tag: v1.11.6
+go_tag: 1.19.7.linux-amd64
+
+ansible_ssh_private_key_file: ~/devnet_private.key
+ansible_ssh_common_args: >
+  -o IdentitiesOnly=yes
+  -o StrictHostKeyChecking=no
+  -o ProxyCommand="sh -c \"aws ssm start-session --target %h --document-name AWS-StartSSHSession --parameters 'portNumber=%p'\""
```

Comment:
It seems you need to add --region us-west-2 else it was not working for me since my default is another region.

Reply:
That's a good point. We should probably comment on that in the instructions or add a variable.